app/lib/paper_trail_version.rb

```diff
@@ -0,0 +1,47 @@
+# Extensions to base class of PaperTrail.
+class PaperTrailVersion < PaperTrail::Version
+
+  IRRELEVANT_FIELDS = %w[user_ids updated_by_id pledge_sent_by_id last_edited_by_id identifier updated_at].freeze
```

Comment:
Are these permanently irrelevant, irrelevant until their model is migrated, or a mix?

Reply:
These are only really relevant to the `patient` model, and more particularly the changelog section that gets populated. I think we could strip these if we wanted to.